src/styles/getPopOverVerticalOffset/index.desktop.ts

```diff
@@ -1,13 +1,12 @@
 import CONST from '../../CONST';
+import GetPopOverVerticalOffset from './types';
 
-/**
- * Adds the header padding with vertical offset on desktop
- * @param {Number} vertical
- * @returns {Object}
- */
-export default (vertical) => ({
+// Adds the header padding with vertical offset on desktop
```

Comment:
For better intellisense:
```suggestion
/** Adds the header padding with vertical offset on desktop */
```

Reply:
+1 can you add this?

Reply:
updated